en/02_Development_environment.adoc

```diff
@@ -47,8 +49,9 @@ There are other libraries available for this purpose, like https://www.libsdl.or
 You can find the latest release of GLFW on the http://www.glfw.org/download.html[official website].
 In this tutorial we'll be using the 64-bit binaries, but you can of course also choose to build in 32 bit mode.
 In that case make sure to link with the Vulkan SDK binaries in the `Lib32` directory instead of `Lib`.
-After downloading it, extract the archive to a convenient location.
-I've chosen to create a `Libraries` directory in the Visual Studio directory under documents.
+After downloading it, extract the archive to a convenient location like `C:\Vulkan-Tutorial`.
+
+Extract glfw there so that you have a path called `C:\Vulkan-Tutorial\glfw` with contents similar to this:
```

Comment:
Maybe also think about showing how to use CMake and a packaging service such as vcpkg in windows apt in linux.  This way, the project will be cleaner and easier to work with in other projects.

Reply:
That's the plan for the long run ;)